### PR TITLE
Ci setup

### LINKED ISSUE #10: Add codecov
Need to add codecov for automatically generated code coverage reports.

### PR DESCRIPTION
**Pull request by somexpert**
This branch should set up the Travis and Codecov integrations. Will fix #11 and will fix #10.

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review. First time for me trying to set up the codecov thing, so I'm not quite sure I did that correctly.

File: tox.ini [hunk 1]
```diff
@@ -3,7 +3,7 @@ addopts = --reuse-db
 DJANGO_SETTINGS_MODULE = tests.settings
 
 [tox]
-envlist = py27-django16, flake8
+envlist = py27-django16
```
Comment (damonkelley) [inline tox.ini]: Was Django 18 and Django master left out intentionally?

Comment (somexpert) [inline tox.ini]: For now, yes. I don't really want to be seeing all the failures on those locally when I run the tests. Once I'm done with the tests and start working on the upgrades and changes, then those will be enabled.

**Comment by damonkelley** [comment]
:+1:

**Pull request merged**